src/platforms/common/configuration/releases.mdx

```diff
@@ -82,6 +82,12 @@ After configuring your SDK, you can install a repository integration or manually
 
 Monitor the [health of releases](/product/releases/health/) by observing user adoption, usage of the application, percentage of [crashes](/product/releases/health/#crash), and [session data](/product/releases/health/#session). Release health will provide insight into the impact of crashes and bugs as it relates to user experience, and reveal trends with each new issue through the [Release Details](/product/releases/release-details/) graphs and filters.
 
+<Note>
+
+Crash and app hang detection are not available for watchOS.
```

Comment:
```suggestion
Crash and app hang detection is not available for watchOS.
```

Reply:
It's two different features, maybe it should say "Crash **reporting** and app hang detection are not available for watchOS"?

Reply:
Yes, that sounds good!

Reply:
```suggestion 
Crash reporting and app hang detection are not available for watchOS.
```